src/axom/slic/streams/LumberjackStream.hpp

```diff
@@ -109,6 +109,15 @@ class LumberjackStream : public LogStream
    */
   virtual void write();
 
+  /*!
+   * \brief Confirms that abort flag(s) was set among all ranks.
```

Comment:
Style suggestion, for the sake of clarity (take as you please):
```suggestion
   * \brief Confirms that abort flag(s) was set on one or more ranks.
```